src/app_service/service/community/dto/community.nim

```diff
@@ -34,6 +34,13 @@ type
     Members,
     ControlNodeUptime
 
+type
+  CommunityMemberState* {.pure.} = enum
```

Comment:
I think this name is not accurate. Either it needs to say it's for "negative" states or we add the `joined` and `joinedPending` states too so that it represents the name correctly.

Reply:
renamed, btw i'm considering to unite it with request to join state to have only one enum responsible for membership